ingestion_server/env.docker

```diff
@@ -9,3 +9,5 @@ UPSTREAM_DB_PORT="5432"
 
 LOCK_PATH="/worker_state/lock"
 SHELF_PATH="/worker_state/db"
+
+INDEXER_WORKER_HOST="indexer_worker"
```

Comment:
Will this require any changes in the infrastructure repo?

Reply:
No it should not. It's a change specific to the local Docker setup.